test/functional/RasapiTest/build.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0"?>
+
+<!--
+  Copyright (c) 2016, 2021 IBM Corp. and others
+
+  This program and the accompanying materials are made available under
+  the terms of the Eclipse Public License 2.0 which accompanies this
+  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+  or the Apache License, Version 2.0 which accompanies this distribution and
+  is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+  This Source Code may also be made available under the following
+  Secondary Licenses when the conditions for such availability set
+  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+  General Public License, version 2 with the GNU Classpath
+  Exception [1] and GNU General Public License, version 2 with the
+  OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+<project name="Build script for com.ibm.jvm.ras.tests" default="build" basedir=".">
+	
+	<property name="build.jar.tests" value="com.ibm.jvm.ras.tests.jar" />
+	<property name="DEST" value="${BUILD_ROOT}/functional/RasapiTest" />
+	<property name="src.level" value="1.7"/>
+	<property name="target.level" value="1.7"/>
+	<property name="source.dir" location="./src" />
+	<property name="dummy.source.dir" location="./dummy_src" />
+	<property name="build.dir" location="./build" />
+	<property name="dist.dir" location="./dist" />
+	<property name="src" location="." />
+	<property name="LIB" value="junit4"/>
+	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
+	
+	<property name="dir.bootcompiler" value="/bluebird/tools/boot-compiler"/>
+	<property name="build.compiler.java" value="${dir.bootcompiler}/sdk70/latest/bin/javac" />
```

Comment:
`build.compiler.java` is not used. Please remove `build.compiler.java`  and `dir.bootcompiler`